bokehjs/src/coffee/models/glyphs/patches.coffee

```diff
@@ -77,8 +77,8 @@ class PatchesView extends Glyph.View
   _render: (ctx, indices, {sxs, sys}) ->
     # @sxss and @syss are used by _hit_point and sxc, syc
     # This is the earliest we can build them, and only build them once
```

Comment:
Won't this get built every render call, not just once? I'm also a little worried that since render is called with different sxs and sys for selection and non-selection glyphs, that one or the other "wins" by overwriting the other, and that `@renderer.sxss` thus only has the subset of patches for whichever render was called last